code_search/kubeflow/environments/pipeline/globals.libsonnet

```diff
@@ -4,4 +4,5 @@
   // e.g. "dataflowImage", "trainerCpuImage", "trainerGpuImage",
   experiment: "pipeline",
   waitUntilFinish: "true",
+  readGithubDatasetForFunctionEmbedding: "true",
```

Comment:
Should this be a global? It only applies to a single component so shouldn't it be a component parameter?

Reply:
revert this flag. thx